packages/plugin-ext/src/main/browser/comments/comment-thread-widget.tsx

```diff
@@ -498,6 +499,16 @@ export class ReviewComment<P extends ReviewComment.Props = ReviewComment.Props>
             </div>
         </div>;
     }
+    localeDate(timestamp: string | undefined): string {
```

Comment:
Does this method require `public` visibility, or can be reduced to `protected`?

Reply:
Visibility reduced to protected, no need to expose.